Java/cp/Rabin_Karp.java

```diff
@@ -0,0 +1,34 @@
+import java.util.*;
+class  Rabin_Karp
+{
+    public static void main(String[] args)
+ {
+   String actualText,patternToBeSearched;
+   Scanner sc=new Scanner(System.in);
+   System.out.println("Enter actual text");
+   actualText=sc.nextLine();
+   System.out.println("Enter pattern to be searched");
+   patternToBeSearched=sc.nextLine();
+   int indexI,indexJ,locationIndex=-1,count=0;

```

Comment:
Try following [Google java style guide](https://google.github.io/styleguide/javaguide.html) to write readable and clean code. For now, add space before and after each operator.